### PR TITLE
mhddfs: init at 0.1.39

### PR DESCRIPTION
**Pull request by makefu**

### CODE REVIEW

File: pkgs/tools/filesystems/mhddfs/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  buildInputs = [ fuse pkgconfig attr uthash ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp mhddfs $out/bin/
+  '';
+
+  meta = {
+    homepage = http://mhddfs.uvw.ru/;
+    description = "combines a several mount points into the single one";
```
Comment (jagajaga) [inline pkgs/tools/filesystems/mhddfs/default.nix]: Please start with a capital letter.

File: pkgs/tools/filesystems/mhddfs/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, fuse, pkgconfig, attr, uthash }:
+
+stdenv.mkDerivation rec {
+  name = "mhddfs";
```
Comment (pSub) [inline pkgs/tools/filesystems/mhddfs/default.nix]: Please append the version to the name.

**Comment by makefu** [comment]
travis seemed to have died due to "error: getting status of ‘/home/travis/build/NixOS/nixpkgs/pkgs/os-specific/linux/miraclecast’: No such file or directory"

@pSub for #11008 the version is also missing in the name, should i create a new PR for this?

**Comment by jagajaga** [comment]
@makefu yeah, please make a PR for https://github.com/NixOS/nixpkgs/pull/11008, that was my mistake that I haven't seen that there is no version in the name. Thank you!

**Comment by pSub** [comment]
For #11008 this is fixed in 59bb76e8945841bffd40c286794a2022748a3910.

**Comment by makefu** [comment]
@psub i fixed the version issue for this package saturday, thanks for fixing the snapraid package :)

**Comment by pSub** [comment]
Thanks. This PR looks good now. I will merge.

**Pull request merged**